neuralop/losses/data_losses.py

```diff
@@ -402,6 +401,77 @@ def __call__(self, y_pred, y, h=None, **kwargs):
             normalization constant for reduction, by default None
         """
         return self.rel(y_pred, y, h=h)
+
+class PointwiseQuantileLoss(object):
+    def __init__(self, alpha, reduce_dims = 0, reductions='sum'):
+        """PointwiseQuantileLoss computes Quantile Loss described in [1]
+
+        Parameters
+        ----------
+        alpha : float
+            value, between 0 and 1, of the proportion of points
+            in the output domain expected to fall within predicted quantiles
+        reduce_dims : int, optional
+            dimensions to reduce when summing, by default 0
+            This loss was formulated for functions with a co-domain in R, 
+            so for now only 0 is supported
+        reductions : str, optional
+            how to apply reduction (sum or mean), by default 'sum'
+
+        References
+        1. Calibrated Uncertainty Quantification for Operator Learning via Conformal Prediction
```

Comment:
The references should be a proper section and the reference and actual link. Let's please try to use correct rst syntax whenever possible so it gets rendered correctly.